src/builder.rs

```diff
@@ -103,28 +103,14 @@ impl Document {
 
         let current_layer = self.get_current_layer();
 
-        let mut font_size = 13.0;
-        let mut line_height = 15.0;
-
-        // Rudimentary text scaling to get the Ascii Armor text to fit
-        if pem.lines().count() > 42 {
-            font_size = 6.5;
-            line_height = 7.0;
-        } else if pem.lines().count() > 39 {
-            font_size = 7.0;
-            line_height = 8.0;
-        } else if pem.lines().count() > 27 {
-            font_size = 8.0;
-            line_height = 9.0;
-        } else if pem.lines().count() > 22 {
-            font_size = 10.0;
-            line_height = 12.0;
-        }
+        let font_size = 8.0;
```

Comment:
This simplifies the logic of creating the pem text by using a single value and centering the text.

Tested with `Hello World` in `in.txt` with:
```bash
cargo run  -- in.txt --force --no-footer
```
Tested with 127KiB of data in `in.txt` with:
```bash
cargo run --features=compression -- in.txt --force --no-footer --title=MistyHills --compress
```